KNOWN_ISSUES.md

```diff
@@ -26,21 +22,18 @@ Please report any other issue you may encounter by using the app help form.
 
 **Issues** caused by lower-level bugs that will be **harder to fix or circumvent**:
 
-* Replacing a directory with a file of the same name won't work (same when
-  replacing a file with a directory of the same name).
```

Comment:
Still unsupported:

- https://github.com/cozy-labs/cozy-desktop/blob/master/test/scenarios/replace_dir_with_file/scenario.js#L6
- https://github.com/cozy-labs/cozy-desktop/blob/master/test/scenarios/replace_file_with_dir/scenario.js#L6

Reply:
My bad. I thought we had fixed that.